aries-core/src/js/components/core/Popover/Popover.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable react/prop-types */
+import { Box, Button, Drop } from 'grommet';
+import { Close } from 'grommet-icons';
+
+export const Popover = ({
+  a11yTitle,
+  title,
+  children,
+  footer,
+  targetRef,
```

Comment:
Should we call this `target` similar to Layer? https://v2.grommet.io/layer#target

Reply:
changed